WalletWasabi.Gui/Controls/WalletExplorer/CoinListViewModel.cs

```diff
@@ -583,5 +554,10 @@ public void OnClose()
 			Disposables?.Dispose();
 			Disposables = null;
 		}
+
+		internal void InvokeDequeueCoinsPressed(CoinViewModel target)
+		{
+			DequeueCoinsPressed.Invoke(target, EventArgs.Empty);
```

Comment:
Note: Im not sure why an event is needed here! We should think about getting rid of this in future (I know you didnt add it here)

Reply:
Yeah, i think it is needed for `CoinJoinViewModel` to know which coin is dequeue but yeah, not sure how to simplify that